### DISCUSSION TITLE
kube-bench:v0.6.8 is showing a CVE-2016-4074 which is Fixed By jq 1.6_rc1-r0

### DISCUSSION THREAD

**Discussion by sbollap1** [aquasecurity/kube-bench#1204] [Questions and Help] [answered]
Hello folks, I am trying to push this image to a repo aquasec/kube-bench:v0.6.8, but unable to, seeing this error

Error: Gate failed  due to a Vendor rated High CVE-2016-4074 which is Fixed By jq 1.6_rc1-r0. I wanted to know if there is a Dockerfile for this image that I can use to update this particular jq version. appreciate any directions

**Comment by mozillazg** [comment, marked as answer]
@sbollap1 The Dockerfile is here: https://github.com/aquasecurity/kube-bench/blob/main/Dockerfile
